addons/project/static/src/components/subtask_kanban_list/subtask_kanban_list.scss

```diff
@@ -14,7 +14,20 @@
         }
     }
 
+    .subtask_list_row {
+        .o_field_widget.o_field_many2many_tags_avatar .o_quick_assign {
+            visibility: hidden;
+        }
+    }
+
     .subtask_list_row:hover {
         background-color: mix($table-color, $table-bg, percentage($table-hover-bg-factor + 0.1));
+        @include media-breakpoint-up(sm) {
+            &:hover {
```

Comment:
Are these really needed ? can't we just do what's below ?

Reply:
@pkyriakou , Yes we need this because we will not add it here then it will create a conflict between task hovering and subtask hovering.
If we are hovering on a task then it should only be visible on a task but not on the subtask.

Reply:
Alright, thank you for the clarification :muscle: